src/components/molecules/VenueJoinButton/VenueJoinButton.scss

```diff
@@ -0,0 +1,4 @@
+.venue-join-modal__title {
```

Comment:
The filename is `VenueJoinButton`, but this is `.venue-join-modal`, we should align these to be the same.

Reply:
Also, following BEM, we should have a 'block' class here, as well as the 'element' (as defined).

I would tend to write the styles for this as:

```scss
.venue-join-modal {
  &__title {
    text-align: center;
    padding-bottom: 20px;
  }
}
```